jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/complex/FormFields.java

```diff
@@ -0,0 +1,316 @@
+package com.epam.jdi.light.angular.elements.complex;
+
+import com.epam.jdi.light.angular.asserts.*;
+import com.epam.jdi.light.angular.elements.composite.*;
+import com.epam.jdi.light.common.*;
+import com.epam.jdi.light.elements.base.*;
+import com.epam.jdi.light.elements.common.*;
+import com.epam.jdi.light.elements.complex.*;
+import com.epam.jdi.light.logger.*;
+import com.epam.jdi.tools.map.*;
+import org.openqa.selenium.*;
+
+import java.util.*;
+
+import static com.epam.jdi.light.common.TextTypes.*;
+import static com.epam.jdi.light.driver.WebDriverFactory.*;
+
```

Comment:
/**
 * To see an example of FormField web element please visit https://material.angular.io/components/form-field/overview#form-field-appearance-variants
 */

add comment before class
2.  If it's not critical, change the name of the element class for FormField (single)